core/app/layout/pkgdata.go

```diff
@@ -70,12 +75,28 @@ func layout(ctx context.Context) (out FileLayout) {
 				return binLayout{dir}
 			}
 		}
-		// We're possibly dealing with a sparse-build, as used by robot.
-		// gapis always has to exist.
-		if _, err := file.FindExecutable(dir.Join("gapis").System()); err == nil {
-			return binLayout{dir}
+	}
+
+	if *runfiles != "" {
+		if layout, err := RunfilesLayout(file.Abs(*runfiles)); err == nil {
+			return layout
+		}
+	}
+
+	if path := file.Abs(file.ExecutablePath().System() + ".runfiles_manifest"); path.Exists() {
+		if layout, err := RunfilesLayout(path); err == nil {
+			return layout
 		}
 	}
+
+	// We're possibly dealing with a sparse-build, as used by robot.
+	// gapis always has to exist.
+	// TODO: this is kind of dumb, since gapis will always find itself.
```

Comment:
Assuming this is the `gapis` process... What about `gapit`?

Reply:
yes, for gapit it sort of makes sense, because it might find gapis. Gapis will, of course always find itself in its own directory. That is why I moved it to be last in priority.